### DISCUSSION TITLE
Looks like Github For you page style is broken

### DISCUSSION THREAD

**Discussion by SanjeebLama** [community/community#54997] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="1105" alt="image" src="https://github.com/community/community/assets/51410633/127ec270-b736-4ee8-8189-215d1254f12e">

[a comment by an automated account is not shown]

**Comment by talkshrey** [comment]
I am facing a similar problem

**Comment by Irie1** [comment]
I can't disable the Global Navigation feature(or whatever causes the new layout, I believe its that) in any browser. Thought it was adblock(ublock), then thought it was brave shield.. or brave all together. Same for chrome. Haven't tried FF.

[a comment by an automated account is not shown]

**Comment by Shitongshuang** [comment]
怎么样才可以结束进程

**Comment by SanjeebLama** [comment]
Closing this discussion because it has been fixed.

**Reply by j4k0xb** [reply]
its still broken for me
